lib/iris/plot.py

```diff
@@ -662,7 +663,18 @@ def _ensure_cartopy_axes_and_determine_kwargs(x_coord, y_coord, kwargs):
     # Ensure the current axes are a cartopy.mpl.GeoAxes instance.
     axes = kwargs.get('axes')
     if axes is None:
-        _replace_axes_with_cartopy_axes(cartopy_proj)
+        if (isinstance(cs, iris.coord_systems.RotatedGeogCS) and
```

Comment:
What bout an `iris.coord_system.GeogCS` coordinate system??

Are we at risk of producing unexpected bespoke behaviour here?
Is there a means to provide a more generalised solution?